crypto/cms/cms_lib.c

```diff
@@ -603,16 +603,10 @@ STACK_OF(X509) *CMS_get1_certs(CMS_ContentInfo *cms)
     for (i = 0; i < sk_CMS_CertificateChoices_num(*pcerts); i++) {
         cch = sk_CMS_CertificateChoices_value(*pcerts, i);
         if (cch->type == 0) {
-            if (!certs) {
-                certs = sk_X509_new_null();
-                if (!certs)
-                    return NULL;
-            }
-            if (!sk_X509_push(certs, cch->d.certificate)) {
+            if (!X509_add_cert_new(&certs, cch->d.certificate, 1)) {
```

Comment:
`X509_ADD_FLAG_UP_REF` instead of 1 here? There are a number of similar instances of X509_add_cert_new() where the last param is 1 or 0 instead of the flag name.

Reply:
Oops, yes, I forgot updating the arguments of `X509_add_cert_new()`.
By design and coincidence, the semantics was still correct, but now the respective macros are used.